### DISCUSSION TITLE
Theory of Rational Numbers?

### DISCUSSION THREAD

**Discussion by guluchen** [cvc5/cvc5#11301] [Q&A]
Hi, I want to ask is there a support to declare rational number variables?
I might be able to simulate it by declaring two integer variables and do a division, but it would be much less efficient.

**Comment by ajreynol** [comment]
Sorry for the delay in reply.  At the moment we don't have plans to support a theory of rationals. 

At the moment the best way to do this would be to introduce the division as you mention.
